src/ParallelHashMap.h

```diff
@@ -273,8 +270,8 @@ void FixedHashMap<K,V>::insert(K key, V value)
  *      key was already present in map.
  */
 template <class K, class V>
-int FixedHashMap<K,V>::insert_and_get_count(K key, V value)
-{
+int FixedHashMap<K,V>::insert_and_get_count(K key, V value) {
+
   /* get hash into table using fast modulus */
   size_t key_hash = std::hash<K>()(key) & (_M-1);
 
```

Comment:
According to the file history, Tim He did. Since this PR modifies some things in `ParallelHashMap.h` I was just looking for anything that should be updated in this file. Can you just remove the braces?